.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,6 +13,11 @@ If this PR depends on other PRs, please list them below this comment,
 using the following format:
 - [ ] depends on: #abc [optional extra text]
 - [ ] depends on: #xyz [optional extra text]
+
+To indicate co-authors, include lines at the bottom of the commit message 
+using the format:
```

Comment:
```suggestion
(that is, before the `---`) using the format:
```
Otherwise LGTM.
bors d+

Reply:
Ah, I didn't see Eric's comment before I wrote this, but he makes a good point too.